### DISCUSSION TITLE
zsh-npm-scripts-autocomplete question

### DISCUSSION THREAD

**Discussion by ochilliam** [nicknisi/dotfiles#224] [General]
Hi @nicknisi 
One thing I've noticed is the [zsh-npm-scripts-autocomplete]( https://github.com/grigorii-zander/zsh-npm-scripts-autocomplete) doesn't work for me. 
when I write npm [TAB] nothing is suggested. looking into dotfiles I can see it's listed in zshrc file as plugin but couldn't figure out why it's not working.

Could you please help?

**Comment by ochilliam** [comment]
While this discussion exists, I just remembered about Eslint, prettier  autosave + formatting. 
Any suggestion how to make it work like vscode? 

I've added this line but it's slow. @nicknisi 
```
vim.api.nvim_create_autocmd("BufWritePre", {
  pattern = "*.ts,*.tsx,*.js,*.jsx",
  command = "EslintFixAll",
})
```

**Reply by nicknisi** [reply]
This should already be set up via [conform](https://github.com/stevearc/conform.nvim). Based on the screenshot below (the tmux status bar) are you on the latest version?

https://github.com/nicknisi/dotfiles/blob/bc52de68d32e966326eb49bdbc99645883523c14/config/nvim/lua/plugins/lsp/init.lua#L28-L37

**Reply by ochilliam** [reply]
understood. thanks. 

about tmux status bar, I'm on latest branch and I've only changed the theme. no matter what  theme I choose, they all have this icon+text collapse (see image) issue.

those must be nerd-icons but why they can't center properly or keep their width and height while there's text, I could not figure out.

and any idea about [zsh-npm-scripts-autocomplete](https://github.com/grigorii-zander/zsh-npm-scripts-autocomplete)?

**Comment by ochilliam** [comment]
Um, one more thing that's bothering me, the tmux status line, the icons aren't centered. same as new page name
<img width="788" alt="image" src="https://github.com/nicknisi/dotfiles/assets/65443737/ed0f5a32-ed68-4759-a77b-86fc6a1d2f50">

@nicknisi I'd appreciate any help.
